redhat-developer/app-services-cli
# fix(service-registry create): improve error handling

Improve error handling of service registry create command.

Closes #1714 

### Verification Steps

1. Try creating a registry instance with same name when user has quota:
```
rhoas service-registry create --name test
```
It should throw the error: `Instance with name "test" already exists`

2. Try creating a registry instance when user has exhausted quota:
It should throw the error: `Instance cannot be created. You have reached the limit of service registry instances per user.`

### Type of change
